frontend/appflowy_flutter/lib/workspace/presentation/home/menu/menu_user.dart

```diff
@@ -60,6 +83,18 @@ class MenuUser extends StatelessWidget {
     );
   }
 
+  Color _generateRandomNameColor(String name) {
+    int hashCode = name.hashCode;
+    Random random = Random(hashCode);
+    Color color = Color(random.nextInt(0xffffffff));
+    double luminance = color.computeLuminance();
+    // Checking if the color is too light and returning the darker version
+    if (luminance > 0.5) {
+      return color.withOpacity(0.8);
+    }
+    return color.withOpacity(1.0);
```

Comment:
Making it opaque is not the same as changing the alpha. Not sure if modifying the opacity is a good idea considering themes, as it might look different depending on the background color.

Consider it.